packages/@lwc/babel-plugin-component/src/decorators/wire/validate.ts

```diff
@@ -25,60 +25,67 @@ function validateWireId(id: NodePath | undefined, path: NodePath, state: LwcBabe
         );
     }
 
-    const isMemberExpression = id.isMemberExpression();
+    let adapter: NodePath<types.Identifier>;
```

Comment:
I never like using `let`, but I reckon it's the most idiomatic way without changing things here too much.